README.md

```diff
@@ -29,10 +30,57 @@ Additionally, it is possible to build RPMs, which install the connectors as serv
 Building the RPMs is possible only within an RPM-based system. A [Vagrantfile](https://github.com/vmware/connectors-workspace-one/blob/master/Vagrantfile)   
 is provided for this purpose.   
 
-### Installation
-
-Please see README files within the [individual connectors](https://github.com/vmware/connectors-workspace-one/tree/master/connectors) for details on how to install the RPMs.
-
+### Installing
+
+Each connector has its own RPM. For example, `jira-connector-1.0.0.noarch.rpm`.
+
+The first step is to use the RPM to install the connector as a service. For example:
+```
+yum install jira-connector-1.0.0.noarch.rpm 
+```
+Before the service can be run, some configuration is required. The connectors authenticate requests expecting an access token from VMware IDM. These tokens are JWTs whose signatures must be verified using a public key. This public key is acquired from a URL that is supplied to the connector via a new configuration file. 
+
+For example, for the Jira connector:
+```
+echo "security.oauth2.resource.jwt.key-uri=https://acme.vmwareidentity.com/SAAS/API/1.0/REST/auth/token?attribute=publicKey&format=pem" \
+> /etc/opt/vmware/connectors/jira/application.properties
+```
+The hostname of the URL will vary depending on your IDM tenant.
+
+The configuration file created above must be part of the `roswell` user and group. Again, using Jira as an example:
+```
+chown roswell:roswell /etc/opt/vmware/connectors/jira/application.properties
+```
+The connector being a Spring Boot application, many other configuration options are available&mdash;for example, `server.port`. Please see the Spring Boot documentation for more details.
```

Comment:
Is `&mdash;` leftoever from an xml or html or something?

Reply:
It's the easiest way I know of to display an em dash.

Reply:
Ah, I didn't realize that was markdown (it looked like xml/html).

Reply:
It is HTML, as there's currently no way to do it with markdown. I guess there's some kind of pass-through capability at play here.